plonk/benches/bench.rs

```diff
@@ -177,7 +299,28 @@ fn bench_batch_verify() {
     plonk_batch_verify_bench!(BW6_761, Fr761, PlonkType::UltraPlonk, 1000);
 }
 
+fn bench_intense() {
+    let mut f = File::create(format!(
+        "../target/{}-threads.txt",
+        rayon::current_num_threads()
+    ))
+    .expect("Unable to create file");
```

Comment:
```suggestion
        let mut path = PathBuf::new();
        path.push(env!("CARGO_MANIFEST_DIR"));
        path.push(format!("target/plonkbench/{}-threads", rayon::current_num_threads()));
        path.set_extension("txt");
        
        let mut f = OpenOptions::new()
            .write(true)
            .create(true)
            .truncate(true)
            .open(path.clone())
            .unwrap();
```

You can try using something like this to overwrite the file instead of appending to it in case the file already exists (which [`File::create`](https://doc.rust-lang.org/std/fs/struct.File.html#method.create) does)